test/AspNet.Security.OAuth.Providers.Tests/Coinbase/CoinbaseTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace AspNet.Security.OAuth.Coinbase
+{
+    public class CoinbaseTests : OAuthTests<CoinbaseAuthenticationOptions>
+    {
+        public CoinbaseTests(ITestOutputHelper outputHelper)
+        {
+            OutputHelper = outputHelper;
+        }
+
+        public override string DefaultScheme => CoinbaseAuthenticationDefaults.AuthenticationScheme;
+
+        protected internal override void RegisterAuthentication(AuthenticationBuilder builder)
+        {
+            builder.AddCoinbase(options => ConfigureDefaults(builder, options));
+        }
+
+        [Theory]
```

Comment:
Just noticed there's no test case for the email claim.

Reply:
I'll add the test for this myself post-merge so I can get the next release shipped.